distribution/changelog.txt

```diff
@@ -10,6 +10,7 @@
 - Feature: [#13376] Open custom window at specified tab.
 - Feature: [#13384] [Plugin] Expose all TileElement data.
 - Feature: [#13398] Add pause button to the Track Designer.
+- Feature: [#13436] Fix MacOS Icon for Big Sur
```

Comment:
Still needs changing the wording.

Reply:
Oh, should the wording be the bug?  I thought this was good wording.

Reply:
This is not a bug, but an enhancement/feature. But otherwise: yes, for bugs the changelog lists the bug in question and not the solution.

BTW, I requested the changed wording [here](https://github.com/OpenRCT2/OpenRCT2/pull/14268#discussion_r589270581), but you closed that without actually applying it.

Reply:
Oh I'm sorry I just didn't even notice the suggestion, I'll put that on right now!